lib/studies/rally/login.dart

```diff
@@ -397,7 +397,7 @@ class _FilledButton extends StatelessWidget {
       style: TextButton.styleFrom(
         backgroundColor: RallyColors.buttonColor,
         primary: Colors.black,
-        padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 24),
+        padding: const EdgeInsets.symmetric(vertical: 14, horizontal: 24),
```

Comment:
As mentioned in the ticket, the smaller Rally login button on desktop is probably due to [`VisualDensity`](https://api.flutter.dev/flutter/material/VisualDensity-class.html) being set to `adaptivePlatformDensity` and thus `compact` for Rally in desktop.

With your change the Button would become bigger on Android and iPhone phones. We will want to set the `visualDensity` to [`VisualDensity.standard`](https://api.flutter.dev/flutter/material/VisualDensity/standard-constant.html) instead.

To do this update the `ThemeData` for Rally: https://github.com/flutter/gallery/blob/2900cb93aeed58c953bda570774894ec7d30b08c/lib/studies/rally/app.dart#L58